Automattic/woocommerce-payments
# Transactions List - indicate when a transaction is expected to be included in a future deposit

Fixes #7876

#### Changes proposed in this Pull Request

In the Transactions List, for transactions that are not included in a pending/paid deposit, render "Future deposit" and an info icon in the "Deposit date" column.

Clicking this icon will present a tooltip with the message:

```
This transaction will be included in a future deposit. Once created, the estimated deposit date will be shown here.
```

This will render for all transaction types except `financing_*`. [See list of transaction types](https://github.com/Automattic/woocommerce-payments/blob/ceda996a805c0f556b55dbd10b665682f184112f/client/transactions/strings.ts#L9-L24).


<img width="1179" alt="image" src="https://github.com/Automattic/woocommerce-payments/assets/3147296/bbec8464-d56c-4f08-9a45-1983f6437dcf">


#### Testing instructions





* View the Transactions List
* Ensure non-financing transactions that are not included in previous deposits show the tooltip icon.
* Clicking on the tooltip icon should show the message defined above.

-------------------


**Post merge**
